pkg/scheduler/apis/config/v1beta1/defaults.go

```diff
@@ -275,6 +275,15 @@ func SetDefaults_VolumeBindingArgs(obj *v1beta1.VolumeBindingArgs) {
 	}
 }
 
+func SetDefaults_NodeResourcesBalancedAllocationArgs(obj *v1beta1.NodeResourcesBalancedAllocationArgs) {
+	if len(obj.Resources) == 0 {
+		obj.Resources = append(obj.Resources, []v1beta1.WeightlessResource{
```

Comment:
nit:
```golang
append(obj.Resources,
    {Name: "foo"},
    {Name: "bar"},
)
```

In other words: no need to create a slice that you just unwrap with `...`

Reply:
Done